cypress/integration/VisitorCanCreateAPostUnsuccessfuly.spec.js

```diff
@@ -0,0 +1,34 @@
+describe('Visitor can', () => {
+
+  beforeEach(function () {
+    cy.server();
+    cy.route({
+      method: 'POST',
+      url: 'http://localhost:3002/api/v1/posts',
+      response: 'fixture:create_post_no_success.json',
+      status: 422
```

Comment:
See my comment on the fixture file

Reply:
`Status: 422` not needed since we are not testing for status

Reply:
Actually, I was wrong: status i needed in the integration file since that is what triggers the error message.. But removed from the fixture file